src/autocomplete/databases/yql/helpers.ts

```diff
@@ -295,6 +326,36 @@ function getColumnsSuggestions({
     );
 }
 
+function getAllColumnsSuggestions(props: GetParticularSuggestionProps): boolean | undefined {
```

Comment:
same here, let's just return boolean

Reply:
also I'd consider renaming all these functions that return boolean instead of suggestions
`get` is not correct here, as we are not getting any suggestions
can name them `shouldSuggestAllColumns` or sth similar